paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerSheet.kt

```diff
@@ -30,23 +31,27 @@ import javax.inject.Inject
  * [CustomerAdapter].
  */
 @ExperimentalCustomerSheetApi
-class CustomerSheet @Inject internal constructor(
+class CustomerSheet internal constructor(
```

Comment:
Looks like there's some modules and components that can be cleaned up. Mind doing that in a follow up? Looks like they were already unused before this change.